netlify/configs/amp.dev/netlify.toml

```diff
@@ -0,0 +1,754 @@
+[build]
+  base = "../../../"
+  # Directory to change to before starting a build.
+  # This is where we will look for package.json/.nvmrc/etc.
+  # If not set, defaults to the root directory.
+
+  publish = "dist/pages"
+
+[[headers]]
+  for = "/*"
+
+  [headers.values]
+    Content-Security-Policy-Report-Only="default-src * data: blob:;worker-src 'self' blob:;script-src blob: 'unsafe-inline' https://cdn.ampproject.org/v0.js https://cdn.ampproject.org/v0.mjs https://cdn.ampproject.org/v0/ https://cdn.ampproject.org/sw/ https://cdn.ampproject.org/viewer/ https://cdn.ampproject.org/rtv/ https://www.googletagmanager.com/gtag/js https://playground.amp.dev/ https://preview.amp.dev/ https://go.amp.dev/ https://log.amp.dev/;object-src 'none';style-src 'unsafe-inline' https://cdn.ampproject.org/rtv/ https://playground.amp.dev/ https://preview.amp.dev/ https://go.amp.dev/ https://log.amp.dev/;report-uri /csp-report"
+    X-Content-Type-Options="nosniff"
+    X-XSS-Protection="1; mode=block"
+
+[[headers]]
+  for = "/who-am-i"
+
+  [headers.values]
+    Content-Type="application/json"
+
+[[redirects]]
+  from = "/documentation/components/*"
+  to = "/documentation/components/:format/:splat?redirected"
+  query = {format = ":format"}
+  status = 301
+  force=true
+
+[[redirects]]
+  from = "/documentation/guides-and-tutorials/*"
+  to = "/documentation/guides-and-tutorials/:format/:splat?redirected"
+  query = {format = ":format"}
+  status = 301
+  force=true
+
+
+[[redirects]]
+  from = "/documentation/examples/*"
+  to = "/documentation/examples/:format/:splat?redirected"
+  query = {format = ":format"}
+  status = 301
+  force=true
+
+[[redirects]]
+  from = "/documentation/courses/*"
+  to = "/documentation/courses/:level/:splat/?redirected"
+  status = 301
+  force=true
+  query = {level = ":level"}
+
+#################
+# Imported from platform/config/go-links.yaml
```

Comment:
Is this manually maintained?

Reply:
It was added manually, yes. would be trivial to add it during the build step from the contents of the go.yml file if you wanted that ability however.

Reply:
that'd be good